### PR TITLE
Pep export button

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: src/components/table/ExportXlsx.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import Api from 'api';
+import { EXPORT_PEP_XLSX_LIMIT } from 'const/const';
+import YesNoModal from 'components/modals/YesNoModal';
+import toast from 'utils/toast';
+
+
+const ExportXlsx = (props) => {
+  const { t } = useTranslation();
+  const { exportUrl, params, count } = props;
+  const ExportXlsxModalRef = useRef();
```
Comment (roman-tiukh) [inline src/components/table/ExportXlsx.jsx:15]:
```suggestion
  const exportXlsxModalRef = useRef();
```

File: src/components/table/ExportXlsx.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  const detail = 'detail';
```
Comment (roman-tiukh) [inline src/components/table/ExportXlsx.jsx:16]: what is this? `const detail = 'detail';`

File: src/components/table/ExportXlsx.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  let element;
+
+  const openExportXlsxModal = () => {
+    ExportXlsxModalRef.current.show();
+  };
```
Comment (roman-tiukh) [inline src/components/table/ExportXlsx.jsx:21]: this function can be inline

File: src/components/table/ExportXlsx.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        position="bottom"
+        arrow={false}
+        content={t('exportTooltipContent', { limit: EXPORT_PEP_XLSX_LIMIT })}
+        className="flex mx-2 cursor-default text-gray-500"
+      >
+        <Download className="w-5 h-5 mr-1 color-gray-500" />
+        Excel
+      </Tooltip>
+    );
+  }
```
Comment (roman-tiukh) [inline src/components/table/ExportXlsx.jsx:56]: use `return` for this...

File: src/components/table/Table.jsx [hunk 5]
```diff
@@ -116,6 +117,15 @@ const Table = (props) => {
         {/*    count: tc.count,*/}
         {/*  })}*/}
         {/*</div>*/}
+        { exportUrl && (
+          <div className="mr-6">
+            <ExportXlsx
+              exportUrl={exportUrl}
+              params={tc.getExportUrlParams()}
```
Comment (roman-tiukh) [inline src/components/table/Table.jsx:124]: just use `getUrlParams`, don't duplicate methods
```suggestion
              params={tc.getUrlParams()}
```

File: src/locales/uk/translation.json [hunk 6]
```diff
@@ -85,6 +85,8 @@
   "division": "Розділ",
   "downloadContract": "Завантажити договір",
   "downloadPEPDatabase": "Завантаження всієї бази РЕР",
+  "downloadXlsx": "Зараз буде вигрузка даних в форматі .xlsx",
+  "downloadXlsxConfirmation": "Після завершення на Ваш email буде відправлено лист з посиланням на скачування цього файлу. Продовжити?",
```
Comment (roman-tiukh) [inline src/locales/uk/translation.json:89]: put **all** messages what have special values to other object:
```suggestion
  "export": {
    "downloadXlsx": "Зараз буде вигрузка даних в форматі .xlsx",
    "downloadXlsxConfirmation": "Після завершення на Ваш email буде відправлено лист з посиланням на скачування цього файлу. Продовжити?",
  },
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/components/table/ExportXlsx.jsx (hunk 1 above)
Comment (Iva-khar) [reply src/components/table/ExportXlsx.jsx:15]: done

File: src/components/table/ExportXlsx.jsx (hunk 2 above)
Comment (Iva-khar) [reply src/components/table/ExportXlsx.jsx:16]: deleted

File: src/components/table/ExportXlsx.jsx (hunk 3 above)
Comment (Iva-khar) [reply src/components/table/ExportXlsx.jsx:21]: done

File: src/components/table/ExportXlsx.jsx (hunk 4 above)
Comment (Iva-khar) [reply src/components/table/ExportXlsx.jsx:56]: done

File: src/components/table/Table.jsx (hunk 5 above)
Comment (Iva-khar) [reply src/components/table/Table.jsx:124]: `getUrlParams()` contains default pagination. I need params for export without pagination.

> just use `getUrlParams`, don't duplicate methods

done

File: src/locales/uk/translation.json (hunk 6 above)
Comment (Iva-khar) [reply src/locales/uk/translation.json:89]: done

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**